frontend-react/src/pages/home/Home.tsx

```diff
@@ -45,6 +46,29 @@ export const Home = () => {
                     );
                 })}
 
+                <div className="tablet:margin-bottom-8">
+                    <h2 className="font-sans-lg tablet:font-sans-xl margin-top-0 tablet:margin-bottom-0">
+                        How it works
+                    </h2>
+                    <p className="usa-intro margin-top-1 text-base">
+                        Our open-source platform aggregates and securely
+                        delivers health care data test results from
+                        organizations and testing facilities directly to public
+                        health entities.
+                    </p>
+                    <img
+                        src="/assets/img/ReportStreamDiagram_2022.png"
+                        alt="Organizations and testing facilities submit data, which ReportStream validates, augments and standardizes. Then, the data is routed as ReportStream determines the destination by subscription management. After this, it is transformed and transported to public health agencies in the desired data model and batch size."
+                    />
+                    <p>
```

Comment:
@jpandersen87 Can you apply `class="usa-prose"` to this section to improve legibility & consistency with the text styling in the above section?

Reply:
Good eye on noticing what is actually a site-wide issue due to USWDS. USWDS DOES NOT apply site-wide baseline styles. Their `usa-prose` also only applies to direct children elements, and not to all elements inside. We should async/sync on this sometime, as I have plans to create a site-wide override. In the meantime, will def add the `usa-prose` here for this PR.